keras_nlp/models/gpt2/gpt2_causal_lm.py

```diff
@@ -291,9 +298,14 @@ def make_generate_function(self):
         if self.generate_function is not None:
             return self.generate_function
 
-        def generate_function(prompt, input_mask, min_length):
+        def generate_function(inputs):
```

Comment:
Please add a comment with the expect structured for `inputs`

Reply:
made it flat!